README.md

```diff
@@ -32,6 +32,20 @@ $ jshint app/assets/javascripts
 ## Deployment
 
 ### Development Mode
+
+You need multiple tools for the full development cycle. Here is the rundown for Ubuntu 14.04 LTS as a reference. If you run into errors with `bundle`, the most likely reason is that the database servers are not properly installed with dev headers/libs.
+
+* nodejs
```

Comment:
Oh, I'm on that Ubuntu version, but don't have nodejs installed (`dpkg -l | grep node` -> empty output). Sure it's needed?